package.json

```
@@ -30,6 +30,7 @@
     "@vue/cli-service": "^4.4.0",
     "@vue/ui": "^0.11.6",
     "@walletconnect/web3-provider": "^1.1.1-alpha.0",
+    "assets": "github:balancer-labs/assets#fabien/registry",
```

```suggestion
    "assets": "github:balancer-labs/assets#master",
```
https://github.com/balancer-labs/assets/pull/8 is merged

---

Thx will update package-lock